docs/csharp/quick-starts/list-collection.yml

```diff
@@ -0,0 +1,167 @@
+    that you've learned in earlier quick starts to loop through the list.
+
+    The code to display names makes use of **interpolated strings**.  When you precede a `string` with the `$` character, you can embed C# code in the string declaration. The actual string replaces that C# code with the value it generates. In this example, it replaces the `{name.ToUpper()}` with each name, converted to capital letters, because you called the <xref:System.String.ToUpper%2A> method.
+
+    Let's keep exploring.
+    
+    > [!NOTE]
+    > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
+
+- title: Modify list contents
+  durationInMinutes: 3
+  content: |
+    The collection you created using the <xref:System.Collections.Generic.List%601> type. This type stores sequences of elements. You specify the type of the elements between the angle brackets.
+    
+    One important aspect of this <xref:System.Collections.Generic.List%601> type is that it can grow or shrink, enabling you to add or remove elements. Add this code below the code you've already written:
+
+    ```csharp
+    Console.WriteLine();
+    names.Add("Maria");
+    names.Add("Bill");
+    names.Remove("Ana");
+    foreach (var name in names)
+    {
+      Console.WriteLine($"Hello {name.ToUpper()}!");
+    }
+    ```
+
+    You've added two more names to the end of the list. You've also removed one as well.
+    
+    The <xref:System.Collections.Generic.List%601> enables you to reference individual items by **index** as well. You access items using the `[` and `]` tokens. Add this code to the bottom and try it:
+
+    ```csharp
+    Console.WriteLine($"My name is {names[0]}");
+    Console.WriteLine($"I've added {names[2]} and {names[3]} to the list");
+    ```
+
+    You're not allowed to access past the end of the list. You can check how long the list is using the `<xref:System.Collections.Generic.List%601.Count%2A>` property:
+
+    ```csharp
+    Console.WriteLine($"The list has {names.Count} people in it");
+    ```
+    
+    Click **Run** again to see the results. In C#, indices start at 0, so the largest valid index is one less than the number of items in the list.
+
+    > [!NOTE]
+    > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
+
+- title: Search and sort lists
+  durationInMinutes: 5
+  content: |
+    Our samples use relatively small lists, but your applications may often create lists with many more elements, sometimes numbering in the thousands. To find elements in these larger collections, you need to search the list for different items. The <xref:System.Collections.Generic.List%601.IndexOf%2A> method searches for an item and returns the index of the item. Try this to see how it works:
```

Comment:
tell them to keep the existing code and add this to the bottom otherwise they might think it's new code